core-header-panel.html

```diff
@@ -298,32 +287,19 @@
 
         _atTop: {
           value: true
-        },
-
-        _dropShadowClass: {
-          computed: 'computeDropShadowClass(_atTop, mode, shadow)'
-        },
-
-        _mainContainerClass: {
-          computed: 'computeMainContainerClass(mode)'
         }
-
       },
 
-      computeDropShadowClass: function(_atTop, mode, shadow) {
+      _computeDropShadowHidden: function(_atTop, mode, shadow) {
         var needsShadow = _atTop &&
           MODE_CONFIGS.shadowMode[this.mode] ||
           MODE_CONFIGS.noShadow[this.mode];
```

Comment:
It makes sense to me to use the arguments passed to computed property methods even if the values happen to be set on the instance before the method is called. The latter statement seems like an implementation detail especially if this computed method API is going to continue taking the new values as arguments.